babel.config.js

```
@@ -1,32 +1,36 @@
+// The source code including full typescript support is available at: 
```

ESLint

[error] Delete `·`

(prettier/prettier)